services/datamanager/builtin/builtin_sync_test.go

```diff
@@ -172,9 +172,9 @@ func TestSyncEnabled(t *testing.T) {
 			b, err := New(context.Background(), deps, config, dataSyncServiceClientConstructor, tc.connStateConstructor, logger)
 			test.That(t, err, test.ShouldBeNil)
 			defer b.Close(context.Background())
-			t.Log("waiting for data capture to write a data capture file")
+			t.Logf("waiting for data capture to write a data capture file %s", time.Now())
```

Comment:
is adding `time.Now()` to logs necessary? don't we log with timestamps?

Reply:
the test logger doesn't write timestamps no. I added them when I was debugging why logs were timing out after reducing the max sync threads.